uwblueprint/paramedics-react
# Updating frontend queries & pin types

[Notion ticket](https://www.notion.so/uwblueprintexecs/396370c5e09f4c28bc7f455a7e84c908?v=3751aa62a2594c74b7e0ecb363304c24&p=380f161e725846019b6d5002e046ff7c)

[Figma link](https://www.figma.com/file/IpOPyv79gBwEohvOJvPV6f/%F0%9F%9A%91-Designs-for-Development-S20-F20?node-id=3958%3A55234)

## Changes
- Updated the front-end location pin queries to match the backend changes listed in [this PR](https://github.com/uwblueprint/paramedics-web/pull/83)
- Updated to use pin icons from Figma, allowing us to visually identify different pin types

## Screenshots
Orange pins represent `OTHER` pins, while the green pin represents `EVENT` pins. 
<img width="1422" alt="Screen Shot 2020-12-13 at 2 17 55 PM" src="https://user-images.githubusercontent.com/21224282/102024222-007f0580-3d4e-11eb-9977-00e2676cdd36.png">

## Testing
**Ensure that you are on the [correct backend branch](https://github.com/uwblueprint/paramedics-web/pull/83)**
- Add a pin and make sure an orange marker opens
- Add different types of pins on the backend and see the different pins on your frontend
    - I had caching issues with this where my pins would not update even though I changed my backend. If you run into this issue, please refer to [Michelle's comment](https://uwblueprint.slack.com/archives/G0149QDHPEV/p1607028941006800). 

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
